plugins/pantheon-files-ctags/plugin.vala

```diff
@@ -328,180 +328,159 @@ public class Marlin.Plugins.CTags : Marlin.Plugins.Base {
         }
     }
 
-    private class ColorWidget : Gtk.MenuItem {
-        private new bool has_focus;
-        private const int BUTTON_WIDTH = 10;
-        private const int BUTTON_HEIGHT = 10;
-        /* Set start margin to match other menuitems. Is there a way to determine
-         * this programmatically?  */
-        private const int MARGIN_START = 27;
-        private const int SPACING = 15;
+    private class ColorButton : Gtk.Image {
+        private const int BUTTON_WIDTH = 16;
+        private const int BUTTON_HEIGHT = 16;
+        private string color_name = "";
+        private string palette_name = "";
+
+        public ColorButton (int id, string color_name, string palette_name) {
+            this.palette_name = palette_name;
+            this.color_name = color_name;
+
+            var css_provider = new Gtk.CssProvider ();
+
+            string style = """
+            .color-button {
+                border-bottom-left-radius: 16px;
+                border-top-left-radius: 16px;
+                border-top-right-radius: 16px;
+                border-bottom-right-radius: 16px;
+                text-shadow: 1px 1px transparent;
+                padding: 0;
+            }
+            .color-%s {
+                background-color: @%s\_300;
+                border: 1px solid @%s\_500;
+            }
+            .color-%s:hover {
+                background-color: @%s\_100;
+                transition: all 100ms ease-out;
+            }
+            .nohover:hover {
+                background: @bg_color;
+            }
+            """.printf(color_name, palette_name, palette_name, color_name, palette_name);
 
+            try {
+                css_provider.load_from_data(style, -1);
```

Comment:
Please make sure to use spaces before `()`. There are a few instances of this in this PR
```suggestion
                css_provider.load_from_data (style, -1);
```